src/data/map.js

```diff
@@ -134,7 +137,7 @@ const getPokemon = async (minLat, maxLat, minLon, maxLon, showPVP, showIV, updat
             ) ${sqlAndIv} ${sqlOrIv}
         ) AND (
             (form = 0 ${sqlExcludeIvPokemon}) OR form NOT IN (0 ${sqlExcludeIvForms})
-        ) ${sqlIncludeIv}
+        ) ${sqlIncludeIv} ${sqlIncludeOnlyVerifiedTimers}
```

Comment:
Wrong placement of SQL.

Reply:
I disagree

Reply:
If you don't believe me go test it.

Reply:
Add a per-Pokemon IV and now it will filter only that Pokemon. I don't care anyways since I think this should be removed.